### ISSUE TITLE
propagate-changes creates two change sets

### ISSUE THREAD

**Issue by sebastiankirsch** [tenpureto/tenpureto#120]
Hey, we're trying to build a validation pipeline for feature changes.
Currently, we can't even execute our approach manually, b/c `tenpureto template propagate-changes` seems to create duplicate changesets.

This is our scenario: our template has features α, a & b. Both a & b are based on α.
To verify changes we wont to do in in α, we do:

    git clone https://github.com/org/project
    cd project
    git checkout a
    git checkout b
    git checkout α
    echo "bar" > foo.txt
    git commit -a -m 'changing α'
    tenpureto template propagate-changes --template $(pwd)

Now what we see is something like this:
```
Changes for a:
    ae7dd71 Merge "α" into "a"
    8319bf5 changing α
    
Changes for a:
    ae7dd71 Merge "α" into "a"
    8319bf5 changing α
    
Changes for b:
    7d8274c Merge "α" into "b"
    8319bf5 changing α
    
Changes for a:
    7d8274c Merge "α" into "b"
    8319bf5 changing α

Do you want to push to "a", "a", "b", "b" (y/n)? [n] y
"git" command failed with exit code 1: error: dst ref refs/heads/a receives from more than one src
error: failed to push some refs to '/tmp/project/'
```

When running with the `--pull-request` option, we see only 2 commit sets to push:
```
Changes for a:
    ae7dd71 Merge "α" into "a"
    8319bf5 changing α
    
Changes for b:
    7d8274c Merge "α" into "b"
    8319bf5 changing α
    
Do you want to push to "a", "b" (y/n)? [n] y
```

Is this a bug in tenpureto or are we approaching this the wrong way?

**Comment by rtimush** [comment]
This looks like a bug in tenpureto. The `--pull-request` option will not work for local repositories, but the regular mode should. I will check what's happening.

**Comment by rtimush** [comment]
Can you please check if the binaries built [here](https://github.com/tenpureto/tenpureto/actions/runs/350445813) work for you? If yes, I will make a release.

**Comment by sebastiankirsch** [comment]
Works for me.
Thanks!

**Comment by rtimush** [comment]
Released as [0.6.1](https://github.com/tenpureto/tenpureto/releases/tag/v0.6.1)
